plugins/plugin-k8s/package.json

```diff
@@ -32,7 +32,8 @@
     "needle": "2.3.1",
     "parse-duration": "0.1.1",
     "semver": "6.0.0",
-    "tmp": "0.1.0"
+    "tmp": "0.1.0",
+    "uuid": "3.3.2"
```

Comment:
uuid should not be the k8s plugin dependency, since it is only used for k8s test, and packages/test already has uuid as dependency. I put it here because running tests in headless build throws error "Cannot find module 'uuid/v4'", and I haven't figure out why.